lib/jellyfish/component/file.ex

```diff
@@ -33,7 +33,8 @@ defmodule Jellyfish.Component.File do
 
       {:ok, %{endpoint: endpoint_spec, properties: %{}}}
     else
-      {:error, _reason} = error -> error
+      {:error, _reason} = error ->
```

Comment:
undo <-> reverse

Reply:
It's triggered by adding another case in the `else`

Reply:
😢